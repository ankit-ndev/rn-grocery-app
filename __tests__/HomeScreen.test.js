import React from 'react';
import renderer from 'react-test-renderer';
import Home from '../src/component/HomeScreen';

test('renders correctly', () => {
  const tree = renderer.create(<Home />).toJSON();
  expect(tree).toMatchSnapshot();
});