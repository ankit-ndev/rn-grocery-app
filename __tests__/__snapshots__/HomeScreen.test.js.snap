// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "flex": 1,
      "flexDirection": "row",
      "justifyContent": "center",
    }
  }
>
  <View
    accessible={true}
    focusable={true}
    onClick={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "borderRadius": 8,
        "padding": 16,
      }
    }
  >
    <View
      style={
        Object {
          "backgroundColor": "#73c6b6",
          "height": 50,
          "justifyContent": "center",
          "width": 150,
        }
      }
    >
      <Text
        style={
          Object {
            "color": "white",
            "fontSize": 19,
            "textAlign": "center",
          }
        }
      >
        VIEW LIST
      </Text>
    </View>
  </View>
</View>
`;
